Sources/SwiftDocC/Model/Rendering/Symbol/AvailabilityRenderMetadataItem.swift

```diff
@@ -163,3 +163,27 @@ public struct AvailabilityRenderItem: Codable, Hashable, Equatable {
         self.isBeta = isBeta
     }
 }
+
+extension Array where Array.Element == AvailabilityRenderItem {
+    /// Determines whether all platforms in the array are in beta.
+    ///
+    /// This property returns `true` if every availability item in the array has its `isBeta` property set to `true`,
+    /// indicating that the symbol is introduced in a beta version across all platforms. If the array is empty,
+    /// this property returns `false`, as an item with no platform availability information is not considered
+    /// to be in beta.
```

Comment:
If this behavior is clarified in the API then this documentation can be simplified to only covering the empty behavior.

Also, there's no need to wrap documentation to a fixed column width. 
```suggestion
    /// If the array is empty, this value is `false`. A symbol without any platform availability information isn't considered to be in beta.
```